### PR TITLE
Channel serializations

### PR DESCRIPTION
**Pull request by velzevur**
[Tags](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aecore/src/aec_object_serialization.erl#L73-L81)

[Create transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_create_tx.erl#L214-L223)

[Deposit transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_deposit_tx.erl#L178-L184)

[Withdraw transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_withdraw_tx.erl#L180-L186)

[Close mutual transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_close_mutual_tx.erl#L188-L194)

[Close solo transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_close_solo_tx.erl#L179-L184)

[Slash transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_slash_tx.erl#L179-L185)

[Settle transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_settle_tx.erl#L208-L214)

[Offchain transaction template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_offchain_tx.erl#L194-L202)

[Channel template](https://github.com/aeternity/epoch/blob/a22ffa239f3ef63126410cadb3bad66f5efed1b0/apps/aechannel/src/aesc_channels.erl#L171-L179)

### CODE REVIEW

File: serializations.md [hunk 1]
```diff
@@ -113,6 +113,16 @@ subsequent sections divided by object.
 | Contract call | 41 |
 | Contract create transaction | 42 |
 | Contract call transaction | 43 |
+| Contract call transaction | 43 |
```
Comment (lucafavatella) [inline serializations.md]: Twice?

**Comment by velzevur** [comment]
Comment addressed

File: serializations.md [hunk 2]
```diff
@@ -361,4 +370,117 @@ purging them from the tree.
...
+, <initiator>        , binary()
+, <responder>        , binary()
+, <initiator_amount> , int()
+, <responder_amount> , int()
+, <updates>          , [{binary(), binary() ,int()}]
+, <state>            , binary()
+]
+```
+
+#### Channel
```
Comment (knarz) [inline serializations.md]: Maybe clarify that this is the object stored in the tree

File: serializations.md [hunk 3]
```diff
@@ -361,4 +370,117 @@ purging them from the tree.
...
+```
+[ <id>              , binary()
+, <initiator>       , binary()
+, <responder>       , binary()
+, <total_amount>    , int()
+, <initiator_amount>, int()
```
[the remark this answers is not in the document]
Comment (velzevur) [reply serializations.md]: `responder_amount` is calculated by using total amount and initiator's amount

**Pull request merged**